articles/encrypting-gdrive-using-rclone/index.md

```diff
@@ -0,0 +1,226 @@
+
+An example command is below:
+
+`rclone copy -v --progress ~/folder/file.txt gdrive:test`
+
+![Rclone Copy Progress](/engineering-education/encrypting-gdrive-using-rclone/rclone-copy.png)
+
+Rclone will start to upload the files/folder you have chosen, and you can see the progress as shown in the image above or through the log file if you have specified one.
+
+### Check the Files Have Transferred
+You can check that the files have transfers either through viewing the log or listing the files in the remote.
+
+If you have set the logging to be output to a file.
+
+Type `cat` and the path to the log file e.g. `cat /opt/rclone.log`.
+
+To check that the files have uploaded in the remote.
+
+Use `rclone lsf remotename:folder` instead.
+
+Here’s an example:  `rclone lsf gdrive:test`
+
+### Mount A Google Drive Folder Using Rclone
+Another great Rclone feature is mounting remotes. Mounting a remote allows you to access your Google Drive files as if they were local ones. This is similar to [Windows’ mapped network drives](https://support.microsoft.com/en-us/help/4026635/windows-map-a-network-drive) but superior because Linux doesn’t classify it as a network drive like Windows does and thus limits functionality.
+
+### Creating the Rclone Mount
+First, we need to create a folder on the local filesystem to store your cloud files. Make sure to give the appropriate permissions so you can access the folder. `mkdir` followed by the folder name will create the folder and `chmod 775` followed by the folder name will apply, read, and write permissions. Here’s an example below:
+
+```bash
+mkdir gdrive
+chmod 755 gdrive
+```
+
+Now that the mounted folder has been created, you can now start the mount command.
+
+The parameters used in the mount command are ones that work best for this particular setup. It’s best to experiment and try out different ones. For example, if you have gigabit upload speed, you might need to limit the maximum data transferred to prevent rate-limiting (Google stops you from transferring data if you do too much in one day).
+
+```bash
+rclone mount --dir-cache-time 96h --cache-info-age 100h --vfs-cache-mode writes --allow-other --log-level DEBUG --log-file /(folder-name)/rclone.log (remote-name): / /(local mount folder)
+```
+
+### Testing the Rclone Mount
+Since the mount command is running, you need to open a new terminal window. Finally, the last step is to check it has worked. The process is the exact same as you would list files in a local folder.
+
+Type `cd (path to the mounted folder)` and then `ls`
+
+```bash
+cd gdrive
+ls
+```
+
+### Encrypt Your Google Drive Remote with Rclone
+Finally, we can encrypt our files. We will set up an encrypted folder to securely store the files that need it whilst still being able to freely access your unencrypted files. Encrypted files can only be decrypted on a computer with Rclone and the encrypted remote.
```

Comment:
>Encrypted files can only be decrypted on a computer with Rclone and the encrypted remote.
It might be better to mention this earlier

Reply:
Thanks. I've mentioned encryption in the intro with a warning and referenced it again in the encryption section.